charts/jx-git-operator/values.yaml

```diff
@@ -24,6 +24,16 @@ image:
 
 replicaCount: 1
 
+versions:
+  # jx is the version of the jx-boot image
+  jx: 3.2.238
```

Comment:
It's probably best to avoid this TBH, the version will become stale unless we keep it up to date using updatebot however doing that will cause a lot of extra releases of the git operator.  I.e. every time jx is released a PR will need to update this repo and ultimately release and cause an upgrade via terraform.  

It's probably better to add this cron job to a repo that is already kept up to date with the latest jx version with update bot like https://github.com/jenkins-x-charts/jxboot-helmfile-resources

This will always have the most recent jx version and also will be upgraded as part of `jx gitops upgrade `https://github.com/jenkins-x/jx/blob/62bc9ddae7feff594ca3de48203b6e036206b9e5/.jx/updatebot.yaml#L54-L60

Reply:
https://github.com/jenkins-x-charts/jxboot-helmfile-resources/pull/393, should we reopen this one then?

Reply:
yeah - I’d not seen the original PR on that repo, yeah you had it right first time.  I grok the original comment of keeping the cron with the operator chart but the image used is jx which means it’s better off outside and pulled into the normal cluster upgrade process